hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -200,13 +200,206 @@
+            - Configure next 6 slots for read commands
+            - Randomize configuration of the remaining 13 cmd info slots.
+            - Issue commands with various opcodes enabled in info slots.
+            - Check proper command propagation.
+            - Disable some cmd info slots.
+            - Check no propagation of disabled commands.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: cmd_read_status
+      desc: '''
+            - Configure proper read status command info slot.
+            - Issue read status command.
+            - Check propagation of read status command.
+            - Initiate response to the read status.
+            - Check proper reception of response.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: cmd_read_jedec
+      desc: '''
+            - Configure proper read jedec command info slot.
+            - Issue read jedec command.
+            - Check propagation of read jedec command.
+            - Initiate response to the read jedec.
+            - Check proper reception of response.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: cmd_read_sfdp
+      desc: '''
+            - Configure proper read sfdp command info slot.
+            - Issue read sfdp command.
+            - Check propagation of read sfdp command.
+            - Initiate response to the read sfdp.
+            - Check proper reception of response.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: cmd_fast_read
+      desc: '''
+            - Configure proper fast read command info slot.
+            - Issue fast read command.
+            - Check propagation of fast read command.
+            - Initiate response to the fast read.
+            - Check proper reception of response.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: flash_cmd_upload
+      desc: '''
+            - Configure flash mode.
+            - Configure cmd info slots.
+            - Set upload to 1 for some of 13 non fixed cmd info slots.
+            - Host should poll busy field status to check if command is done.
+            - Issue next command upload and poll busy status again.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: mailbox_command
+      desc: '''
+            - Configure cmd info slots.
+            - Issue one of predefined read command targeting mailbox space.
+            - Check response to read command.
+            - Check if command is processed internally.
+            - Don't issue command larger than 256 bytes.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: mailbox_cross_outside_command
+      desc: '''
+            - Configure cmd info slots.
+            - Issue one of predefined read command targeting mailbox space.
+            - Command should start inside mailbox space and cross into read space.
+            - Command should be blocked.
+            - Don't issue command larger than 256 bytes.'''
```

Comment:
looks like we can remove this line (same for other places where we put this line) and add a TODO to clarify what data should be returned after crossing boundary (if it's not clear now)